cognite/client/_api/events.py

```diff
@@ -291,7 +292,30 @@ def aggregate(self, filter: Union[EventFilter, Dict] = None) -> List[EventAggreg
                 >>> aggregate_type = c.events.aggregate(filter={"type": "failure"})
         """
 
-        return self._aggregate(filter=filter, cls=EventAggregate)
+        return self._aggregate(filter=filter, cls=AggregateResult)
+
+    def unique_values_aggregate(
+        self, filter: Union[EventFilter, Dict] = None, fields: List[str] = None
+    ) -> List[UniqueValuesAggregateResult]:
+        """`Aggregate unique values for events <https://docs.cognite.com/api/v1/#operation/aggregateEvents>`_
```

Comment:
is having 2 functions for one endpoint not confusing?

Reply:
we decided to go down this route to hide the complexity of the API a bit. If it's a single endpoint, then the user won't know what fields are valid for what aggregation types.